### ISSUE TITLE
Notification when/before entering full channel as admin

### ISSUE THREAD

**Issue by Natenom** [mumble-voip/mumble#1925]
There should be a notification or a "are you sure" dialog before a user with "Write ACL" permission can enter a full room, because as an "admin" he doesn't always know about a limit.

**Comment by mkrautz** [comment]
Either this, or we should make show the user limit in the UI next to the channel name...

    Channel name (10/10)

maybe with the limits more subtle than the name itself...

Though, from experience, if we want this done anytime soon, it should probably just be a dialog that notifies the user before entering. :-)

**Comment by hacst** [comment]
I don't think we should add a dialog. Extending the already existing display for usercount with the limit should be a good enough workaround for any admin concerned about this.

**Comment by hacst** [comment]
Sigh...the way this limit count works differs from the usercount display. For the display we sum the users in all channels including and below the one we want the count for. The limit only considers per channel user-count though :(

**Comment by mkrautz** [comment]
Let's change the way the display works?

**Comment by hacst** [comment]
Then it becomes useless. The original reason we implemented it is so you could see how many users were in channels you didn't expand on big servers.

**Comment by hacst** [comment]
Maybe we can somehow combine it without it becoming completely unparseable...
```(5/∞ 📂10)```
```(5/∞ ▾10)```
```(⏵5/∞ ▾10)```
```(5/∞ 🡦10)```
not exactly found an option standing out yet though...

**Comment by Kissaki** [comment]
As soon as you enter a collapsed channel it expands.
So I would suggest showing the full user number if collapsed, and the user number in *that* channel when expanded.

Although this may be a bit confusing, it is probably the simplest, consistent way to display it.

**Comment by mkrautz** [comment]
Just want to chime in and mention that it was a UI issue that blocked the user local volume changing for a long time.

I wanted the UI to be a simple slider that was visible from a submenu of  the context menu of a user.

But it was hard to get right on all platforms because of sizing issues.

In the end, someone else implemented the feature, but by using a plain old dialog box instead.

I think we should sometimes just pick the easier implementation first, and then consider better solutions once we have something that works for our users (but could work better).
